modules/ROOT/pages/scopes-and-collections-config.adoc

```diff
@@ -0,0 +1,191 @@
+= Scopes and Collections Configuration for Sync Gateway
+:page-aliases: learn/scopes-and-collections-config.adoc
+ifdef::show_edition[:page-edition: {release}]
+ifdef::prerelease[:page-status: {prerelease}]
+:page-role:
+:page-content: conceptual
+:description: pass:q[How to configure Scopes and Collections for Sync Gateway - Examples with descriptions.]
+
+For a successful configuration of your Couchbase Sync Gateway Scopes and Collections, use this helpful example. 
```

Comment:
This section explains how to configure Scopes and Collections for Sync Gateway. It describes configuration options and provides example configuration code.

**Understanding buckets, scopes, and collections**
Each database can contain multiple buckets. Each bucket can hold multiple scopes, and each scope can contain multiple collections.

When you create a bucket, a bucket._default._default element is inherently included. (see the documentation on [Default Scopes and Collections](https://github.com/couchbase/docs-sync-gateway/blob/be0913dac8a8f27a2f06554fd759f3ba9448a8a8/modules/ROOT/pages/server/learn/data/scopes-and-collections.adoc#default-scope-and-collection) in Couchbase Server.) Couchbase creates the default bucket to ensure backward compatibility.

Note: Only Couchbase Server (CBS) allows using the word 'default' in naming conventions. As a result,you may see the term _default.default when creating a new bucket.